lib/dialogs/import/dropzone/index.tsx

```diff
@@ -49,29 +73,42 @@ function ImporterDropzone({
     onDrop,
   });
 
-  useEffect(() => {
-    if (!errorMessage) {
-      return;
-    }
-    const timer = setTimeout(() => setErrorMessage(undefined), 2500);
-    return () => clearTimeout(timer);
-  }, [errorMessage]);
-
-  const text = errorMessage ? errorMessage : 'Drag a file, or click to choose';
-
   const DropzonePlaceholder = () => (
     <Fragment>
-      {errorMessage ? <GridiconWarn /> : <GridiconUpload />}
-      {isDragActive ? 'Drop files here' : text}
+      {errorMessage.length > 0 ? <WarningIcon /> : <CloudIcon />}
+      {isDragActive
+        ? 'Drop files here'
+        : 'Drag and drop to upload files, or click to choose'}
     </Fragment>
   );
 
-  const FileWithIcon = () => (
-    <Fragment>
-      <FileIcon />
-      <span className="importer-dropzone__filename">{acceptedFile}</span>
-    </Fragment>
-  );
+  const FilesWithIcon = () => {
+    const fileList = acceptedFile.map((file: File) => (
+      <li key={file.name}>
+        <FileIcon />
+        {file.name}
+      </li>
+    ));
+
+    const errorList = errorMessage.map((error) => (
+      <li key={error}>
+        <WarningIcon />
+        {error}
+      </li>
+    ));
+
+    return (
+      <Fragment>
+        <div className="accepted-files-header">
+          Import File{acceptedFile.length > 1 ? 's' : ''}
```

Comment:
when dealing with string literals it can be very advantageous to keep them whole, even at the cost of duplicating code. some day someone is going to want to find where in the app it's printing "Import Files" but this wouldn't turn up.

```ts
{acceptedFile.length > 1 ? 'Import Files' : 'Import File'}
```

this is a common issue with translations as well.

Reply:
That makes total sense. In fact, I've run into that in the past when searching for strings in code. I've fixed this now.